README.md

```diff
@@ -48,12 +48,17 @@ Each cloud provider is optional and can be disabled by setting their respective
     * `ac_count` - Number of Alibaba Cloud hosts to run.
     * `ac_type` - Type of host to provision in Alibaba Cloud. (default: `ecs.t5-lc1m1.small`)
     * `ac_data_vol_size` - Size in GiB of extra volume for host. (default: `0`)
+  * __Amazon Web Services__
+    * `aws_count` - Number of Alibaba Cloud hosts to run.
+    * `aws_size` - Size of host to provision in Alibaba Cloud. (default: `ecs.t5-lc1m1.small`)
```

Comment:
Seems to be inconsistent. For other providers we have `*_type` to define type of machine. And it's not `Alibaba Cloud` :)